src/metapath-results.js

```diff
@@ -623,14 +639,15 @@ function makeMetapathsTable(metapaths) {
     metapath_name: 2,
     metapath_abbreviation: 3,
     path_count: 4,
-    p_value: 5,
-    dwpc: 6,
-    dgp_source_degree: 7,
-    dgp_target_degree: 8,
-    dgp_n_dwpcs: 9,
-    dgp_n_nonzero_dwpcs: 10,
-    dgp_nonzero_mean: 11,
-    dgp_nonzero_sd: 12
+    adjusted_p_value: 5,
+    p_value: 6,
+    dwpc: 7,
+    dgp_source_degree: 8,
+    dgp_target_degree: 9,
+    dgp_n_dwpcs: 10,
+    dgp_n_nonzero_dwpcs: 11,
+    dgp_nonzero_mean: 12,
+    dgp_nonzero_sd: 13
```

Comment:
I found this a little tedious and error prone. Maybe we can write something like this:
```
index = 0;
const order = {
  id: ++index,
  metapath_name: ++index,
  ...
  dgp_nonzero_sd: ++index
}
```
Another approach that I can think of is to define `order` as an array of strings:
```['id', 'metapath_name', ..., 'dgp_nonzero_sd']```
and the index of each element will be the order in the sort arrow function.

In both approaches, if decide to switch two elements, we don't have to change the numeric values.

Reply:
Okay, I can change it to the array method. I have this custom sort order in several places. Since this PR is already combining a bunch of small things, I'll change them in all locations here.